sample/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+. /opt/intel/sgxsdk/environment
+
+if [ $# -lt 2 ]; then
+    exit
+fi
+
+# Only the replica node 0 generates keys.yaml
+if [ "$1$2" = "run0" ]; then
+    keytool -o /data/keys.yaml generate -u lib/libusig.signed.so
+else
+    until [ -f /data/keys.yaml ]; do
```

Comment:
Maybe we could use `flock` command to synchronize one-time initialization of `keys.yaml` file. Something like this should work:

```sh
(
  flock 3
  if [ -f /data/keys.yaml ]; then exit; fi
  keytool -o /data/keys.yaml generate -u lib/libusig.signed.so
  rm -f /data/.keys.yaml.lock
) 3>/data/.keys.yaml.lock
```

Reply:
@sergefdrv After some searching, I find an article that says we should not delete the lock file:
https://unix.stackexchange.com/a/368167

I think it is reasonable and plan to add description to remove the lock file manually in the document. What do you think?